docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: "3"
 services:
   odin:
     image: mbround18/odin:latest
+    platform: "linux/amd64"
```

Comment:
i'm on mac m1 with arm.

Reply:
This should only really be in your docker-compose.dev.yaml but have you had success running valheim on m1? weve had lots of issues before with people struggling with it.

Reply:
arrggg no i couldn't dev at all! i should have searched earlier 😢